### PR TITLE
move and rename troubleshooting section

### PR DESCRIPTION
**Pull request by jasonnovichRunAI**

### CODE REVIEW

File: docs/admin/runai-setup/cluster-setup/cluster-install.md [hunk 1]
```diff
@@ -76,38 +76,9 @@ runai-public:
...
-| Waiting to connect | The cluster has never been connected. |
-| Disconnected | There is no communication from the cluster to the Control Plane. This may be due to a network issue. |
-| Missing prerequisites | At least one of the [Mandatory Prerequisites](cluster-prerequisites.md#prerequisites-in-a-nutshell) has not been met. |
-| Service issues | At least one of the *Services* is not working properly. You can view the list of nonfunctioning services for more information |
-| Connected | All services are connected and up and running. |
-
-See the [Troubleshooting guide](../../troubleshooting/cluster-health-check.md#verifying-cluster-health) to help troubleshoot issues in the cluster.
-
-### Troubleshooting
+### Troubleshooting you installation
```
Comment (ozRunAI) [inline docs/admin/runai-setup/cluster-setup/cluster-install.md:81]: Troubleshooting your?

Comment (ozRunAI) [reply docs/admin/runai-setup/cluster-setup/cluster-install.md:81]:
```suggestion
### Troubleshooting installation
```

**Review by ozRunAI — CHANGES_REQUESTED** [review]

File: docs/admin/runai-setup/cluster-setup/cluster-install.md (hunk 1 above)
Comment (jasonnovichRunAI) [reply docs/admin/runai-setup/cluster-setup/cluster-install.md:81]:
```suggestion
### Troubleshooting your installation
```

File: docs/admin/runai-setup/cluster-setup/cluster-install.md [hunk 2]
```diff
@@ -76,38 +76,9 @@ runai-public:
...
-| Waiting to connect | The cluster has never been connected. |
-| Disconnected | There is no communication from the cluster to the Control Plane. This may be due to a network issue. |
-| Missing prerequisites | At least one of the [Mandatory Prerequisites](cluster-prerequisites.md#prerequisites-in-a-nutshell) has not been met. |
-| Service issues | At least one of the *Services* is not working properly. You can view the list of nonfunctioning services for more information |
-| Connected | All services are connected and up and running. |
-
-See the [Troubleshooting guide](../../troubleshooting/cluster-health-check.md#verifying-cluster-health) to help troubleshoot issues in the cluster.
-
-### Troubleshooting
+### Troubleshooting your installation
```
Comment (ozRunAI) [inline docs/admin/runai-setup/cluster-setup/cluster-install.md:81]:
```suggestion
### Troubleshooting installation
```

Comment (ozRunAI) [inline docs/admin/runai-setup/cluster-setup/cluster-install.md:81]:
```suggestion
### Troubleshooting the installation
```

**Review by ozRunAI — CHANGES_REQUESTED** [review]

**Review by ozRunAI — APPROVED** [review]

**Pull request merged**